src/java/org/apache/cassandra/tools/nodetool/stats/CompactionHistoryHolder.java

```diff
@@ -83,6 +85,7 @@ private HashMap<String, Object> getAllAsMap()
             compaction.put("bytes_in", this.bytesIn);
             compaction.put("bytes_out", this.bytesOut);
             compaction.put("rows_merged", this.rowMerged);
+            compaction.put("compaction_type", this.compactionType);
```

Comment:
@Maxwell-Guo you have not changed name of this property to `compactionProperties`.

Reply:
ok